test/bench/document_test.ts

```diff
@@ -28,6 +28,7 @@ const tests = [
         assert.equal('{}', doc.toSortedJSON());
         assert.equal(doc.getCheckpoint(), InitialCheckpoint);
         assert.isFalse(doc.hasLocalChanges());
+        doc.garbageCollect(MaxTimeTicket);
```

Comment:
Is there any reason to run the garbage collector on every test?

Reply:
I added it to remove any unnecessary leftover elements.
I ran the garbage collector, and the memory usage seemed to have decreased, so I pushed it here.
But when I ran the garbage collector and checked the number of elements removed, the result was 0.
Sorry, it seems unnecessary code, I'll delete it.